cookbook/docs/contribute.rst

```diff
@@ -27,50 +26,56 @@ Below you’ll find instructions that will hopefully guide you through how to co
+    2. Package the examples by running ``pyflyte --pkgs core package --image core:v1 -f``.
+    3. Register the examples by running ``flytectl register files --archive -p flytesnacks -d development --archive flyte-package.tgz --version v1``.
+    4. Visit https://localhost:30081/console to view the Flyte console, which consists of the examples present in the ``flytesnacks/cookbook/core`` directory.
+    5. To fetch new dependencies and rebuild the image, run 
+        ``flytectl sandbox exec -- docker build . --tag "core:v2" -f core/Dockerfile``, 
+        ``pyflyte --pkgs core package --image core:v2 -f``, and 
+        ``flytectl register files --archive -p flytesnacks -d development --archive flyte-package.tgz --version v2``.
+    6. Refer to `Build & Deploy Your Application “Fast”er! <https://docs.flyte.org/en/latest/getting_started_iterate.html#bonus-build-deploy-your-application-fast-er>`__ if code in itself is updated and requirements.txt is the same.
 
 📝 Contribute to Documentation
 ==============================
 
-``docs`` folder in ``flytesnacks`` houses the required documentation configuration. All the core, case_studies, and integrations docs are written in the respective README.md and the Python code files. 
+The ``docs`` folder in ``flytesnacks`` houses the required documentation configuration. The core, case_studies, and integrations docs are written in the respective README.md and the Python code files. 
+
+1. README.md file needs to capture the *what*, *why*, and *how*.
 
-1. README.md file needs to capture the *what*, *why*, and *how* 
-    * What is the integration about? Its features, etc.
-    * Why do we need this integration? How is it going to benefit the Flyte users?
-    * Showcase the uniqueness of the integration
-    * How to install the plugin?
+   * What is the integration about? Its features, etc.
+   * Why do we need this integration? How is it going to benefit the Flyte users?
+   * Showcase the uniqueness of the integration
+   * How to install the plugin?
   
-    Refer to any repo in the cookbook directory to understand this better.
+   .. tip::
+      Refer to any repo in the cookbook directory to understand this better.
 
-2. Explain what the code does  
+2. Explain what the code does.
 3. Update `conf.py <https://github.com/flyteorg/flytesnacks/tree/master/cookbook/docs/conf.py>`__ (imagine you have added ``snowflake`` to the ``integrations/external_services`` folder):
    
-      * Add the Python file names to the ``CUSTOM_FILE_SORT_ORDER`` list
-      * Add ``../integrations/external_services/snowflake`` to ``example_dirs``
-      * Add ``auto/integrations/external_services/snowflake`` to ``gallery_dirs``
-4. Verify if the code and documentation look as expected
+   * Add the Python file names to the ``CUSTOM_FILE_SORT_ORDER`` list
+   * Add ``../integrations/external_services/snowflake`` to ``example_dirs``
+   * Add ``auto/integrations/external_services/snowflake`` to ``gallery_dirs``
+
```

Comment:
Add three points here:
4. Add code for panel and TOC in the required RST file.
    <img width="1182" alt="Screenshot 2022-02-23 at 12 33 32 PM" src="https://user-images.githubusercontent.com/27777173/155274419-790f0916-7de0-4abf-9bd0-68c0b52859d3.png">
5. Add ``name`` and ``path`` to ``.github/workflows/ghcr_push.yml`` if you're adding an integration or tutorial.
6. Add an entry to ``cookbook/flyte_tests_manifest.json`` if you're adding an integration or tutorial.